src/adapter/src/coord/appends.rs

```diff
@@ -424,12 +457,28 @@ impl Coordinator {
             updates,
             source: BuiltinTableUpdateSource::DDL,
         });
-        self.group_commit_initiate(None).await;
+        self.group_commit_initiate(None, None).await;
         // Avoid excessive group commits by resetting the periodic table advancement timer. The
         // group commit triggered by above will already advance all tables.
         self.advance_timelines_interval.reset();
     }
 
+    /// Submits a write to be executed during the next group commit and triggers a group commit.
+    ///
+    /// Returns a Future that resolves when the write has completed.
+    pub(crate) fn send_builtin_table_updates_notify<'a>(
+        &'a mut self,
+        updates: Vec<BuiltinTableUpdate>,
+    ) -> BoxFuture<'static, ()> {
+        let (tx, rx) = oneshot::channel();
+        self.pending_writes.push(PendingWriteTxn::System {
+            updates,
+            source: BuiltinTableUpdateSource::Async(tx),
+        });
+        self.trigger_group_commit();
```

Comment:
I think we want to add the following below this line (like in the method above):

```
        // Avoid excessive group commits by resetting the periodic table advancement timer. The
        // group commit triggered by above will already advance all tables.
        self.advance_timelines_interval.reset();
```

Reply:
The call to `trigger_group_commit()` resets the `advance_timelines_interval`